acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -0,0 +1,31 @@
+step "Install PC1 repository" do
+  install_puppetlabs_release_repo(hosts, 'pc1')
+end
+
+step "Install legacy Puppet agents" do
+  default_puppet_version = '1.10.1'
+  puppet_version = ENV['PUPPET_LEGACY_VERSION']
+  if not puppet_version
+    logger.info "PUPPET_LEGACY_VERSION is not set!"
+    logger.info "  using default value of #{default_puppet_version}"
+    puppet_version = default_puppet_version
+  end
+
+  hosts.each { |host| host.install_package('puppet-agent', '', puppet_version) }
+end
+
+step "Install legacy Puppetserver" do
+  master.install_package('puppetserver', '', '2.7.2')
```

Comment:
This is probably a reasonable thing to do.  One other option might be to just configure the PC1 repo and do a puppetserver install, letting whatever "latest" release build is available there be laid down.  That might be advantageous in terms of not wanting/needing to bump this later.  I suppose that would give us a latest released 2.x by default but maybe we'd want to go to a latest released 5.x later on, when that repo configuration has been solidified?  Upgrading from a fixed older-than-latest released version isn't terrible either, though.  Happy to go with whatever seems best to you here.